python_modules/libraries/dagster-dbt/dagster_dbt_tests/core/test_asset_checks.py

```diff
@@ -434,6 +435,35 @@ def test_materialize_checks_no_asset(
     assert len(result.get_asset_observation_events()) == 0
 
 
+def test_get_manifest_and_translator_only_checks(
+    test_asset_checks_manifest: Dict[str, Any],
+    dbt_commands: List[List[str]],
+):
+    my_translator = DagsterDbtTranslator()
+
+    @dbt_assets(manifest=test_asset_checks_manifest, dagster_dbt_translator=my_translator)
+    def my_dbt_assets(context, dbt: DbtCliResource):
+        for dbt_command in dbt_commands:
+            cli_invocation = dbt.cli(dbt_command, context=context)
+            yield from cli_invocation.stream()
```

Comment:
The fixture isn't necessary because no invocation is happening.

```suggestion
def test_get_manifest_and_translator_only_checks(
    test_asset_checks_manifest: Dict[str, Any],
):
    my_translator = DagsterDbtTranslator()

    @dbt_assets(manifest=test_asset_checks_manifest, dagster_dbt_translator=my_translator)
    def my_dbt_assets(context, dbt: DbtCliResource): ...
```

Reply:
👍